### DISCUSSION TITLE
"match required"

### DISCUSSION THREAD

**Discussion by 3dimaging** [kiwanami/emacs-calfw#171] [Q&A]
I tried to import my google calendar to calfw using calfw-ical and calfw-gcal. When I execute the following command:
(cfw:open-ical-calendar "https://calendar.google.com/calendar/ical/.../basic.ics")

I failed to import it. There was a "M-x (match required)" appear before the "   cfw:open-ical-calendar "https://calendar.google.com/calendar/ical/.../basic.ics"    "

**Comment by haji-ali** [comment]
I am gonna assume this is no longer relevant. Let me know if not.
